ktor-server/ktor-server-core/jvm/test/io/ktor/tests/newapi/KtorPluginTest.kt

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.newapi
+
+import io.ktor.application.*
+import io.ktor.application.newapi.KtorPlugin.Companion.createPlugin
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import io.ktor.server.testing.*
+import io.ktor.sessions.*
+import io.ktor.util.*
+import kotlin.test.*
+
+class KtorPluginTest {
+    @Test
+    fun `test empty plugin does not break pipeline`(): Unit = withTestApplication {
+        val plugin = createPlugin("F", createConfiguration = {}) {
```

Comment:
maybe use `{}` as a default argument for  `createConfiguration`?

Reply:
Will create another override with Unit for that, thanks!